src/components/ReportsPage/HeatmapPage/HeatmapFilters.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2023 Cortex Applications, Inc.
```

Comment:
[Non-blocking] Oops! -- this should be updated for 2024 (now that we're halfway through the year already... 😅)